code/__HELPERS/names.dm

```diff
@@ -59,7 +59,7 @@ GLOBAL_VAR(command_name)
 		var/newname
 		if(config && config.station_name)
 			newname = config.station_name
-		else
+		else if(config)

```

Comment:
I did test this and it appears that the station name was fixed? It says FTL13: (random name here) now.